fluentd/configs.d/openshift/output-es-config.conf

```diff
@@ -3,7 +3,7 @@
       host "#{ENV['ES_HOST']}"
       port "#{ENV['ES_PORT']}"
       scheme https
-      index_name ${record['kubernetes_namespace_name']}.${record['kubernetes_namespace_id']}.${Time.at(time).getutc.strftime(@logstash_dateformat)}
+      index_name openshift_${record['kubernetes']['namespace_name']}_${record['kubernetes']['namespace_id']}-vVersion-${Time.at(time).getutc.strftime(@logstash_dateformat)}
       user fluentd
       password changeme
 
```

Comment:
document type depending on whether the log is from container or not needs to be specified here.
not sure what fluentd setting that is. `target_type_key`?